<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImporterOptionsDialog</class>
 <widget class="QDialog" name="ImporterOptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Importer Options</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="9" column="1" colspan="9">
    <widget class="QLineEdit" name="poemTitleLineEdit"/>
   </item>
   <item row="2" column="0" colspan="10">
    <widget class="QTextEdit" name="textEdit_2"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Content Type:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Poems Title pattern:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>File name:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="9">
    <widget class="QPushButton" name="cancelPushButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="10">
    <widget class="QTextEdit" name="textEdit"/>
   </item>
   <item row="5" column="1">
    <widget class="QCheckBox" name="normalTextCheckBox">
     <property name="text">
      <string>Normal Text</string>
     </property>
    </widget>
   </item>
   <item row="0" column="9">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
   <item row="10" column="8">
    <widget class="QPushButton" name="importPushButton">
     <property name="text">
      <string>Import</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="8">
    <widget class="QLineEdit" name="fileNameLineEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QCheckBox" name="classicCheckBox">
     <property name="text">
      <string>Classical Poem</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QCheckBox" name="whiteCheckBox">
     <property name="text">
      <string>White Poem</string>
     </property>
    </widget>
   </item>
   <item row="5" column="4" colspan="6">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="10" column="7">
    <widget class="QPushButton" name="previewPushButton">
     <property name="text">
      <string>Preview</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="7">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
